File: tkpddiscovery/src/main/java/com/tokopedia/discovery/similarsearch/model/SimilarProductsImageSearch.java
package com.tokopedia.discovery.similarsearch.model;

import com.google.gson.annotations.SerializedName;

public class SimilarProductsImageSearch{

	@SerializedName("data")
	private Data data;

	public void setData(Data data){
		this.data = data;
	}

	public Data getData(){
		return data;
	}

	@Override
 	public String toString(){
		return 
			"SimilarProductsImageSearch{" + 
			"data = '" + data + '\'' + 
			"}";
		}
}

File: features/discovery/similarsearch/src/main/java/com/tokopedia/similarsearch/getsimilarproducts/model/SimilarProductsImageSearch.kt
package com.tokopedia.similarsearch.getsimilarproducts.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

internal data class SimilarProductsImageSearch(
        @SerializedName("data")
        @Expose
        val data: Data = Data()
)